custom_dc/quickstart.md

```diff
@@ -19,7 +19,17 @@ This is step 1 of the [recommended workflow](/custom_dc/index.html#workflow).
 
 The instructions in this page use the following setup:
 
-![setup2](/assets/images/custom_dc/customdc_setup2.png)
+![local setup](/assets/images/custom_dc/customdc_setup2.png)
+
+The "data management" Docker container consists of scripts that do the following:
+- Convert custom CSV file data into SQL tables and store them in a data store -- for now, in a local SQLite database
+- Generate ML embeddings for custom data and store them -- for now, in the local file system
+
+The "services" Docker container consists of the following Data Commons components:
+- A [Nginx reverse proxy server](https://www.nginx.com/resources/glossary/reverse-proxy-server/), which routes incoming requests to the web or API server
+- A Python-Flask web server, which handles interactive requests from users
+- An Python-Flask NL server, for generating embeddings and serving natural language queries
```

Comment:
The NL server does not generate embeddings so can remove that part.

Reply:
Done.